src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -2811,6 +2815,29 @@ Bailout:
 
         End Sub
 
+        Friend Shared Function NamedArgumentMustBeTrailing(candidate As Candidate, argumentName As String, paramIndex As Integer) As Boolean
+            Dim correspondingParamIndex As Integer
+            If Not candidate.TryGetNamedParamIndex(argumentName, correspondingParamIndex) Then
+                Return True
+            End If
+
+            ' Named arguments on a ParamArray parameter must be trailing
+            If paramIndex < candidate.ParameterCount AndAlso candidate.Parameters(paramIndex).IsParamArray Then
```

Comment:
>If paramIndex < candidate.ParameterCount AndAlso candidate.Parameters(paramIndex).IsParamArray Then [](start = 12, length = 99)

I am not sure if this check is needed in VB because it never matches named arguments to param arrays.